ephemeral-utils.lua

```diff
@@ -0,0 +1,26 @@
+local _Utils = {}
```

Comment:
why `_` the export object? is that the style in lua now? This looks like lua 5.2+, and it was 5.1 I last used, I'm out of touch I suspect.